R/utilities.R

```diff
@@ -445,16 +449,22 @@ CellCycleScoring <- function(
   object,
   s.features,
   g2m.features,
+  ctrl = NULL,
   set.ident = FALSE,
   ...
 ) {
   name <- 'Cell.Cycle'
   features <- list('S.Score' = s.features, 'G2M.Score' = g2m.features)
+  if(is.null(ctrl)) {
+    ctrl <- min(vapply(X = features, FUN = length, FUN.VALUE = numeric(length = 1)))
+  } else {
+    ctrl <- ctrl
+  }
```

Comment:
The `else` part of this if/else is not necessary. Also, adjust the style for proper spacing and naming of arguments.
```suggestion
  if (is.null(x = ctrl)) {
    ctrl <- min(vapply(X = features, FUN = length, FUN.VALUE = numeric(length = 1)))
  }
```